content/gophercon-2017/writing-a-successful-gophercon-proposal.md

```diff
@@ -0,0 +1,100 @@
++++
+author = ["David Cheney"]
+date = "2017-01-14"
+title = "Writing a successful GopherCon proposal"
+tags = ["gophercon","community"]
+series = ["GopherCon 2017"]
+
++++
+
+[The GopherCon 2017 Call for Proposals](https://www.papercall.io/gophercon2017) has reached the halfway mark so I wanted to pass on advice to potential speakers who are planning to submit a proposal but may be unsure how to go about it.
+
+# Consider your audience.
+
+This year we're asking for proposals for three different kinds of talks; keynotes, tutorials, and workshops.
+The audience for each kind of talk is different.
+
+For keynotes, you'll be speaking to the entire audience, 1500 people whose experience with Go will span everything from beginners, to long time GopherCon attendees, to the Go team themselves.
+This means your topic should be relevant to a wide audience.
+
+This does not mean pitching to the loweset common denominator and trying to sell the audience on the benefits of using Go.
+The audience have already invested time and money in flying to Denver to visit the conference -- they already think Go is pretty great.
+Instead, you want to inspire the audience to convince _other_ people that Go is great, and do that by presenting information that is relevant to Go programmers today.
+
+For tutorials, you have more time, up to 45 minutes, and an audience who are self selecting; they have chosen to come to your talk over the other two talks at the same time.
+You can afford to present in depth on a topic and be prepared to take questions.
+
+For workshops, again you have more time, the whole day, to instruct students who have chosen to attend your specific workshop.
+
+# Keep it relevent
+
+GopherCon is not a general programming conference, the GopherCon audience are expecting to hear about Go.
+If your proposal doesn't mention Go at all, or a reader could substitute the name of another programming language for Go without changing the message of your proposal, you need to dig deeper to find the relevance for a Go programming audience.
+
+# Keep it fresh
+
+Our audience invests a lot in attending the conference and they expect GopherCon to continue its tradition of featuring new speakers and new ideas.
+The reality of our online connected world is many talks are recorded and widely circulated.
+Our audience will not be happy to attend a conference composed of talks they think they've already seen.
+
+If you've presented a talk at a meetup or other conference and plan to propose it for GopherCon, by all means do so, but think about how you can incorporate the feedback you received and how the landscape may have changed since you gave your presentation last.
+
+# Writing a good GopherCon proposal
```

Comment:
How does GopherCon feel about including your job title, name, position, etc in your abstract? I know some people find it necessary to include snippets like 

```
As the CTO of Intellicorp, John Smith has demonstrated a background... 
```

I think maybe we should call out expectations (or at least guidance) around how verbose a potential speaker should (or even is allowed) to be about themselves.